hvz/main/views.py

```diff
@@ -258,6 +259,22 @@ def json_population_time_series(request):
         {'label': 'zombies', 'data': zombie_tally, 'color': 'rgb(0, 128, 0)'},
     ])
 
+def json_get_all_emails(request):
+    """A function that displays all emails.
+    """
+    emails = [p.user.email for p in Player.current_players()]
```

Comment:
We imported all the models in a single module, so it would be `models.Player.current_players()`. Should just work after that.